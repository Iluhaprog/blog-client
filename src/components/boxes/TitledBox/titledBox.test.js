import React from 'react';
import { render } from '@testing-library/react';
import { TitledBox } from '.';

test('Should render titled box', () => {
    const { asFragment } = render(
        <TitledBox title='Test'>
            <div className='item'></div>
            <div className='item'></div>
            <div className='item'></div>
        </TitledBox>
    );
    expect(asFragment()).toMatchSnapshot();
}); 