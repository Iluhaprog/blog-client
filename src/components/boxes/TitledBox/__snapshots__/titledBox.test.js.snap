// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render titled box 1`] = `
<DocumentFragment>
  <div
    class="titled-box"
  >
    <div
      class="column   "
    >
      <h1
        class="titled-box__title"
      >
        Test
      </h1>
      <div
        class="container"
      >
        <div
          class="row row_jc-c row_ai-fs row_w-w"
        >
          <div
            class="item"
          />
          <div
            class="item"
          />
          <div
            class="item"
          />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
